code/modules/mob/mob.dm

```diff
@@ -123,6 +123,123 @@
 	if(bound_overlay)
 		bound_overlay.visible_message(message, self_message, blind_message)
 
+/mob/proc/get_action_string(is_self, var/using_verb, var/object_phrase, var/infix, var/postfix)
+	// use mob gender value because otherwise subject verb agreement gets messed up by disguised mobs
+	// todo: pseudoplural pronouns? we need "Unknown (as David Hasselhoff) spins around." and "You spin around." to both work
```

Comment:
I think technically this is what ey/em is for, but I can see that would be weird/jarring for The Common Player if it just started showing up instead of they/them.